icu4j/tools/utilities-for-cldr/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+* © 2023 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>74.1</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>utilities-for-cldr</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <proj.displayname>Utilities</proj.displayname>
+    <mf.Automatic-Module-Name>com.ibm.icu.utilities</mf.Automatic-Module-Name>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+           <execution>
+               <id>add-source</id>
+               <phase>generate-sources</phase>
+               <goals>
+                  <goal>add-source</goal>
+               </goals>
+               <configuration>
+                  <sources>
+                    <source>../misc/src/main/java/com/ibm/icu/dev/tool/shared</source>
+                    <source>../../main/framework/src/test/java/com/ibm/icu/dev/util</source>
+                    <source>../../main/translit/src/test/java/com/ibm/icu/dev/util</source>
+                  </sources>
+               </configuration>
+           </execution>
+         </executions>
+        </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.ibm.icu</groupId>
+      <artifactId>translit</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+  </dependencies>
+
+  <profiles>
+    <profile>
+      <id>cldr_tools</id>
+      <build>
+        <plugins>
+          <plugin>
+            <!-- Only deploy when the `cldr_tools` profile is active,
+              otherwise it will also be deployed to Maven Central.
+              We can also try to specify `altDeploymentRepository` here.
+            -->
```

Comment:
Is there a reason it can't be deployed to maven central? Then we might be able to have CLDR pull from there also.  Might reconsider this.

Reply:
Until now we only published it to the GitHub repo.
So I didn't want to shake the boat close to a release.

---

I think that using the Nexus snapshot repo would be nice.
It would make it possible to use without the annoying requirement that everyone configures they own access.
(GitHub token, settings.xml, https://cldr.unicode.org/development/maven)

But it is a bit more work to setup the publishing here (setup the Nexus user / password as repo secrets and using them)

Reply:
I thought a bit more about this.
In order to publish to Nexus we need to make sure we never deploy this for non-SNAPSHOT versions.
Otherwise it goes to the public maven repo (no snapshot), and we would be forced to bump the version.
So I would not do this now, I would plan it a bit better.